app/src/features/surveys/components/StudyAreaForm.tsx

```diff
@@ -93,11 +111,26 @@ const StudyAreaForm = () => {
     });
 
     // set values
-    setFieldValue('locations', data);
+    setFieldValue('locations', data, true);
   };
 
   return (
     <form onSubmit={handleSubmit}>
+      <YesNoDialog
+        dialogTitle={`Delete all ${values.locations.length} Study Areas?`}
+        dialogText="Are you sure you want to remove these study areas? This action can be undone by exiting the survey page without saving"
```

Comment:
Kind unsure about the wording here. If the action can be undone, why should the user be prompted?

Reply:
yeah it does feel a little weird but this is just following our delete pattern of prompting the user before anything gets removed.

Reply:
One thought: the original file they uploaded might not be around anymore, so deleting the spatial is maybe slightly more dangerous than changing other survey meta. Would be the same for documents too.